src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php

```diff
@@ -121,45 +122,14 @@ public function build()
                 $this->webspaces[] = $webspace;
 
                 $this->buildPortals($webspace);
-            } catch (\InvalidArgumentException $e) {
-                $this->logger->warning(
-                    sprintf(
-                        'Error: "%s" in "%s". The file has been skipped.',
-                        $e->getMessage(),
-                        $file->getRealPath()
-                    )
-                );
-            } catch (InvalidUrlDefinitionException $e) {
-                $this->logger->warning(
-                    sprintf(
-                        'Error: "%s" in "%s". The file has been skipped.',
-                        $e->getMessage(),
-                        $file->getRealPath()
-                    )
-                );
-            } catch (InvalidCustomUrlException $e) {
-                $this->logger->warning(
-                    sprintf(
-                        'Error: "%s" in "%s". The file has been skipped.',
-                        $e->getMessage(),
-                        $file->getRealPath()
-                    )
-                );
-            } catch (WebspaceLocalizationNotUsedException $e) {
-                $this->logger->warning(
-                    sprintf(
-                        'Error: "%s" in "%s". The file has been skipped.',
-                        $e->getMessage(),
-                        $file->getRealPath()
-                    )
-                );
+            } catch (\Exception $e) {
```

Comment:
I am against catching the base exception class, since also errors not related to webspaces could be swallowed here. Actually, why should we catch an exception here at all? We are rethrowing an exception, making the path to follow unnecessarily harder.